Driverless AI/ES/introducción-a-machine-learning-automático-con-driverless-ai/introducción-a-machine-learning-automático-con-driverless-ai.md

```diff
@@ -455,37 +457,37 @@ Si desea realizar un recorrido rápido por la página **Experiments** (Experimen
-
+ 
 Un conjunto de datos limpios es esencial para la creación de un buen modelo de predicción. El proceso de limpieza de datos debe hacerse con todos los conjuntos de datos para eliminar cualquier conjunto de observaciones no deseadas, errores estructurales, valores atípicos no deseados o datos faltantes.
-
+ 
 6\. Seleccione **Test Dataset** y luego haga clic en **titanic_test**
-
+ 
 ![add-test-set](assets/add-test-set.jpg)
-
-7\. Ahora seleccione la **Target Column**. En nuestro caso, la columna será 'survived' (sobrevivió).
-
+ 
+7\. Ahora seleccione **Target Column**. En nuestro caso, la columna será 'survived' (sobrevivió).
+ 
 ![train-set-drop-name-column](assets/train-set-drop-name-column.jpg)
-
-El atributo "sobrevivió" fue seleccionado porque, como compañía de seguros, queremos saber qué otros atributos pueden contribuir a la supervivencia de los pasajeros a bordo de un barco e incorporarlo a nuestras tarifas de seguro.
-
+ 
+El atributo "sobrevivió" fue seleccionado porque, como empresa de seguros, queremos saber qué otros atributos pueden contribuir a la supervivencia de los pasajeros a bordo de un barco e incorporarlo a nuestras tarifas de seguro.
+ 
 8\. Su página de experimento debería ser similar a la siguiente; Estas son las sugerencias del sistema:
-
+ 
 ![experiment-settings](assets/experiment-settings.jpg)
-
+ 
 *Cosas a tener en cuenta:*
-
+ 
 1. **Training Settings** (Configuración de entrenamiento) - Describe la precisión, el tiempo y la interpretabilidad de su experimento específico. Las perillas en la configuración del experimento son ajustables a medida que los valores cambian el significado de la configuración en el cambio de página inferior izquierdo.
-
+ 
     - [Accuracy](http://docs.h2o.ai/driverless-ai/latest-stable/docs/userguide/experiment-settings.html?highlight=accuracy%20setting#accuracy) (Precisión) - A medida que aumenta la precisión, la IA sin conductor ajusta gradualmente el método para realizar la evolución y el conjunto. Con baja precisión, la IA sin conductor varía características y modelos, pero todos compiten de manera uniforme entre sí. Con mayor precisión, cada modelo principal independiente evolucionará de forma independiente y será parte del conjunto final como un conjunto sobre diferentes modelos principales. Con precisiones más altas, la IA sin conductor evolucionará + tipos de funciones de conjunto, como la codificación de destino, dentro y fuera, que evolucionan independientemente. Finalmente, con las precisiones más altas, la IA sin conductor realiza el seguimiento de modelos y características y combina todas esas variaciones. Cambiar este valor afecta la evolución de la característica y la canalización final.
```

Comment:
"Con baja precisión, la IA sin conductor varía características y modelos"
In the above sentence, are you referring to "Driverless AI" with "la IA sin conductor?"

Reply:
"la IA sin conductor evolucionará **+** tipos de funciones de conjunto"
Is the "+" sign referring to the word "más?"